v1/linkml-schemas/evidence.yaml

```diff
@@ -38,6 +38,32 @@ classes:
       - fraudCheck
       - activityFrom
       - identityCheckLevel
+  SecurityCheckClass:
+    attributes:
+      type:
+        equals_string: SecurityCheck
+    slots:
+      - contraIndicator
+  ContraIndicatorClass:
+    slots:
+      - code
+      - issuanceDate
+      - document
+      - mitigation
```

Comment:
should be also include txn slots?

Reply:
txn is slightly below in the slot_usage (line 54), it is because the txn slot is defined at a global level and we are just re-using.